jaeger-core/src/main/java/com/uber/jaeger/utils/SystemClock.java

```diff
@@ -0,0 +1,48 @@
+/*
+ * Copyright (c) 2016, Uber Technologies, Inc
+ *
+ * Permission is hereby granted, free of charge, to any person obtaining a copy
+ * of this software and associated documentation files (the "Software"), to deal
+ * in the Software without restriction, including without limitation the rights
+ * to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+ * copies of the Software, and to permit persons to whom the Software is
+ * furnished to do so, subject to the following conditions:
+ *
+ * The above copyright notice and this permission notice shall be included in
+ * all copies or substantial portions of the Software.
+ *
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+ * IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+ * FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+ * AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+ * LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+ * OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+ * THE SOFTWARE.
+ */
+package com.uber.jaeger.utils;
+
+/**
+ * Default implementation of a clock that delegates its calls to the system clock.
+ * The microsecond-precision time is simulated by (millis * 1000), therefore
+ * the {@link #isMicrosAccurate()} is false.
+ *
+ * @see System#currentTimeMillis()
+ * @see System#nanoTime()
+ */
+public class SystemClock implements Clock {
+
+    @Override
+    public long currentTimeMicros() {
+        return System.currentTimeMillis() * 1000;
+    }
+
+    @Override
+    public long currentTimeNanos() {
```

Comment:
I'd probably not use the word "Time" here, eventhough nanoTime does. maybe tick or nanoTick? https://google.github.io/guava/releases/19.0/api/docs/com/google/common/base/Ticker.html